X16 Reference - 09 - VERA Programmer's Reference.md

```diff
@@ -569,8 +569,10 @@ Setting **'Layer0 Enable'** / **'Layer1 Enable'** / **'Sprites Enable'** will re
 **'Tile Base Address'** specifies the base address of the tile data. *Note that the register only specifies bits 16:11 of the address, so the address is always aligned to a multiple of 2048 bytes.*
 
 **'H-Scroll'** specifies the horizontal scroll offset. A value between 0 and 4095 can be used. Increasing the value will cause the picture to move left, decreasing will cause the picture to move right.
+Hardware scrolling is only possible in tile mode; in bitmap mode, this register is not functional.
```

Comment:
Perhaps we can add a note here that the high bits are reused for the palette offset, even though it's also mentioned elsewhere.

Reply:
sure, it doesn't hurt to repeat it here

Reply:
pushed text changes